### ISSUE TITLE
P.NATS training database

### ISSUE THREAD

**Issue by chatdeschroedinger** [itu-p1203/open-dataset#2]
Hi,

I am using the [PXNR-No-reference-video-quality-estimation](https://github.com/Ericsson/PXNR-No-reference-video-quality-estimation) and tried to find the P.NATS training database referred to in the report. (P2STR01-P2STR14 and P2SVL0 - P2SVL13).

The video sequences do not seem to be present in this repository. Are they available somewhere else?

Thank you for your response!
Lukas

**Comment by slhck** [comment]
As the material used for generating the test series is mostly copyrighted, we unfortunately cannot supply the actual bitstreams or decoded pixel data, only the bitstream-based metadata. Sorry about that!

**Comment by chatdeschroedinger** [comment]
Ok. I am trying to understand the characteristics of the test set.
May I ask if it is a broad range of video types or specifically video calls or movie video material?

**Comment by slhck** [comment]
I see. The videos in this database were mostly based on the following:

- Publicly available animated open-source movies (Big Buck Bunny, Sintel, Tears of Steel)
- Publicly available video test material (e.g., Netflix El Fuente)
- Movie-like sequences with varying degrees of action
- Documentary-type content
- Sports

But you are actually referring to databases from the P.NATS Phase 2 project, which is *not* what this repo is about. This repo is the set of databases for the P.NATS Phase 1 work item, which led to the creation of the standard series P.1203. P.NATS Phase 2 led to the standard series P.1204 (and the PXNR model you referenced). The difference is that P.1203 used HD material with up to 25 fps, and content lengths of up to 5 minute. Also, only long-duration tests (1-5 min) were done. P.1204 used both short-term test (6-8 seconds) and long-term tests (up to 5 minutes IIRC).

**Comment by slhck** [comment]
For P.NATS Phase 2 the content was very similar, with recordings of gaming added to the set of source material.

Maybe @davidlindero has some further comment.

**Comment by davidlindero** [comment]
@slhck you gave a good summary. the P.1204-databases also contained higher frame rates (60 fps) and resolution (4K)
